utilities/deploy.sh

```diff
@@ -0,0 +1,25 @@
+#!/bin/bash
+
+set -e
+
+SITE_VERSION=$1
+
+if [ -n $SITE_VERSION ]; then
+    echo "First arg (version) not provided, so we're exiting."
+    exit 1
+fi
+
+if [ "${SITE_VERSION##*-}" == "SNAPSHOT" ]; then
```

Comment:
I'm just carrying over what was done in the source script this was copied from. I just want to use as light a touch as possible, because I really just want to move to Gradle. Just filed this: https://github.com/GoogleCloudPlatform/google-cloud-java/issues/1587